apps/meteor/client/views/room/contextualBar/ExportMessages/MailExportForm.tsx

```diff
@@ -159,7 +159,8 @@ const MailExportForm = ({ formId, rid, onCancel, exportOptions }: MailExportForm
 													return undefined;
 												}
 
-												if (additionalEmails !== '' && validateEmail(additionalEmails)) {
+												const emails = additionalEmails?.split(',').map((email) => email.trim());
+												if (additionalEmails !== '' && emails.every((email) => validateEmail(email))) {
```

Comment:
`additionalEmails` is being checked to be empty in L158, and the inverse in L163 

```suggestion
												const emails = additionalEmails?.split(',').map((email) => email.trim());
												if (Array.isArray(emails) && emails.every((email) => validateEmail(email))) {
```

And delete 158 thru 160